manifest/utils.go

```diff
@@ -27,3 +29,22 @@ func (s *mapSlice) UnmarshalYAML(n *yaml.Node) error {
 
 	return nil
 }
+
+func GetNetworkEndpointFromEnvironment(networkName string) string {
+	networkName = mapNetworkNameToCanonicalName(networkName)
+	networkEndpoint := os.Getenv(fmt.Sprintf("SUBSTREAMS_ENDPOINTS_CONFIG_%s", strings.ToUpper(networkName)))
+	return networkEndpoint
+}
+
+func mapNetworkNameToCanonicalName(networkName string) string {
```

Comment:
After thinking about this, it may be a bad idea: the manifest would say something "wrong" like "polygon", and the environment variable SUBSTREAMS_ENDPOINTS_CONFIG_POLYGON would not work... this sucks... Better keep it unchanged...